vulnerabilities/templates/base.html

```diff
@@ -26,12 +27,19 @@
                         <a class ="navbar-item" href = "{% url 'vulnerability_search' %}" >
                         Explore
                         </a>
-                        <a class ="navbar-item" href = "{% url 'vulnerability_create' %}">
-                        Create
-                        </a>
+                        
```

Comment:
That's a minor nit but you want to keep your HTML  properly formatted.
Here this would mean using consistent indentation and spaing (e.g. essentially as if it were Python)

Reply:
Sure thing,  fixing it